hack/verify-boilerplate.sh

```diff
@@ -18,12 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
 boilerDir="${KUBE_ROOT}/hack/boilerplate"
 boiler="${boilerDir}/boilerplate.py"
 
-files_need_boilerplate=($(${boiler} "$@"))
+files_need_boilerplate=()
+while IFS=$'\n' read -r line; do
+  files_need_boilerplate+=( "$line" )
+done < <(${boiler} "$@")
```

Comment:
Maybe $boiler should be in quotes?

Reply:
@fejta 
I checked the details of the failure, which was caused by the conflict of `hack/.shellcheck_failures`.
I've updated the code now.
`${boiler}` doesn't have to be in double quotes, the shellcheck test can pass.

Reply:
yes but it should be quoted, this will not work correctly if there's a space in $BASH_SOURCE (try `mkdir -p "/tmp/very exciting dir/" && mv kubernetes "/tmp/very exciting dir" && cd "/tmp/very exciting dir" && hack/verify-boilerplate.sh`)

Reply:
Thanks a lot @fejta ! 
Updated.